components/kyma-environment-broker/internal/hyperscaler/azure/env_test.go

```diff
@@ -0,0 +1,107 @@
+package azure
+
+import (
+	"testing"
+
```

Comment:
In case you want the testitify suggestion below:

```suggestion

	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/require"

```